Tests/Functional/Controller/responses/navigation__get.json

```
@@ -13,6 +13,7 @@
                 "template": "default",
                 "parent": "@uuid@",
                 "url": "\/test-1",
+                "path": "@string@.optional()",
```

we could check the value of the path like this, but not sure if thats necessary 🙂 

```suggestion
                "path": "@string@.match(\"parent/child\").optional()",
```